backend/src/openbeheer/catalogi/api/views.py

```diff
@@ -80,7 +80,7 @@ def get(self, request: Request, slug: str) -> Response:
                 )
                 # OZ API specs say that is is not required, but VNG specs say it is.
                 # In practice, it is always present.
-                value = catalogue.url
+                value = catalogue.url.split("/")[-1]  # UUID.
```

Comment:
Please use `str.removeprefix` with the service base_url and if you care that it's a uuid, turn it into a UUID and reflect it in the return type.

OZ expects urls. Somewhere at the spot of #71, we'll have to translate this back to whatever we snipped off.